### ISSUE TITLE
Add better team logo support

### ISSUE THREAD

**Issue by splewis** [splewis/get5-web#16]
Potentially the logos from (the MIT licensed) [csgo-data](https://github.com/kokarn/csgo-data) could be used on the site to display next to team names/flags. 

Other ideas:
- adjust the get5_apistats to fetch missing logo files
- allow users to upload logo files (not likely to happen)

**Comment by splewis** [comment]
This is mostly implemented now, but could use a way to upload new files (maybe admin only). Right now you just drag files into get5/static/img/logos

**Comment by TandelK** [comment]
Splewis can you create a space for uploading logo while creating team in same area where we add Steam ID and we can add that as Fast DL as the same folder.  So it can work smooth

**Comment by splewis** [comment]
That is exactly what this issue is for. I'm probably not going to get around to it anytime soon, though.

**Comment by TandelK** [comment]
I found a way around. Just upload the logo to Server and make the server with FastDL of the same /img/static/logos folder and there is one plugin which helps us to download the Logo's Directory from the same folder.  https://forums.alliedmods.net/showthread.php?t=258206

**Comment by Shivam16a** [comment]
Is there anyway to use my custom team logos with already available web panel https://get5.splewis.net/matches

**Comment by splewis** [comment]
No. You need to host your own version to self-upload them. This webpanel is essential a proof-on-concept (I've tried to make this clear on the readme) for others to make their own version, so custom logo support is far from a priority.
